lesson27/src/js/login.js

```diff
@@ -3,6 +3,8 @@ import { togglePassword } from "./module/toggle-password";
 import { Chance } from "chance";
 const chance = new Chance();
 
+if(localStorage.getItem("token")) window.location.href = "./index.html";
+
```

Comment:
The `token` is also checked on the login page. If `token` exists, you will be redirected to the content page.
The `token` is issued only upon successful login and set in local storage.